packages/api-clients/src/selfcareClients.ts

```diff
@@ -1,6 +1,13 @@
 import { ZodiosInstance, ZodiosOptions } from "@zodios/core";
 import { SelfCareConfig } from "pagopa-interop-commons";
-import { selfcareV2ClientApi } from "./index.js";
+import {
+  InstitutionApi,
+  DelegationApi,
+  UserApi,
+  createInstitutionApiClient,
+  createDelegationApiClient,
+  createUserApiClient,
+} from "./generated/selfcareV2ClientApi.js";
```

Comment:
If you do this, no need to change anything in the rest of the file I think (other files here do like this)

```suggestion
import * as selfcareV2ClientApi from "./generated/selfcareV2ClientApi.js";
```